cmd/kyma/alpha/deploy/cmd.go

```diff
@@ -89,10 +92,23 @@ func (cmd *command) Run() error {
 		HelmTimeoutSeconds:            int(cmd.opts.HelmTimeout.Seconds()),
 		BackoffInitialIntervalSeconds: 3,
 		BackoffMaxElapsedTimeSeconds:  60 * 5,
-		Log:                           log.Printf,
+		Log:                           cmd.getLogFunc(),
+	}
+
+	var updateCh chan deployment.ProcessUpdate
+	if cmd.opts.Verbose {
+		defer log.Println("Kyma deployed!")
+	} else {
+		// Start rendering async CLI UI
```

Comment:
Imho the function names are descriptive enough, which makes those comments redundant